packages/io/io/deserializers.js

```diff
@@ -0,0 +1,17 @@
+const amfDeSerializer = require('@jscad/amf-deserializer')
+const dxfDeSerializer = require('@jscad/dxf-deserializer')
+const jsonDeSerializer = require('@jscad/json-deserializer')
+const objDeSerializer = require('@jscad/obj-deserializer')
+// const scadDeSerializer = require('@jscad/scad-deserializer') //FIXME: upgrade, fix before re-enabling
+const stlDeSerializer = require('@jscad/stl-deserializer')
+const svgDeSerializer = require('@jscad/svg-deserializer')
+
+const deserializers = {}
+deserializers[amfDeSerializer.extension] = amfDeSerializer.deserialize
```

Comment:
The repetition here bugs me.  Is something like tis too hacky?
```
const deserializerFormats = ["amf", "dxf", "json", "obj", "stl", "svg"]
const deserializers = {}

deserializerFormats.forEach(format => {
	deserializer = require(`@jscad/${format}-deserializer`);
	deserializers[deserializer.extension] = deserializer.deserialize
})
```

Reply:
Well... yes and no.

The repetition is kind of ugly but only here. And there’s no need to know the extensions, as those are exported by the specific deserializer modules.

The suggestion is good but it means the extensions define the module name. I may have to rename a module.